Data/Scripts/System/Commands/Player/Skills.cs

```diff
@@ -69,6 +69,7 @@ public NewSkillsGump( Mobile from, int origin ) : base( 50, 50 )
 text = text + "Magery - A difficult skill to master as those who seek the power have a long road of knowledge they must traverse. Get a spellbook and search the land for hidden spells you may collect and one day you may become a powerful wizard. Be on the lookout for reagents, as you will need them to summon these magics. Those skilled in Magery also gain a combat benefit with using staves, sceptres, non-spell-imbued wands, and scepters. Mages can never dabble in the spells of Elementalism, as they interfere with each other.<BR><BR>";
 text = text + "Magic Resistance - This not only helps you resist magic used against you, but also enhances many of your other defenses. Some magic cannot be avoided however, but this skill will at least minimize the effects.<BR><BR>";
 text = text + "Marksmanship - This will improve the more you use the appropriate weapons, increasing damage and possibilities to hit with ranged weapons. These are weapons like bows, crossbows, wizard staves, some magic wands, and throwing gloves. You have the ability to throw daggers, stones, harpoons, darts, axes, or shurikens.<BR><BR>";
+text = text + "Meditation - This allows the user to enter a meditative trance to replenish its magical energy. Heavy armor has a negative effect on it's use, prohibiting the user from properly entering a trance.<BR><BR>";
```

Comment:
I think this should be written in a second-person perspective like the other text chunks in here.

```suggestion
text = text + "Meditation - This allows you to enter a meditative trance to replenish your magical energy. Heavy armor has a negative effect on mana absorption, prohibiting you from properly entering a trance.<BR><BR>";
```